inc/3rd-party/hosting/presslabs.php

```diff
@@ -0,0 +1,142 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+if ( defined( 'PL_INSTANCE_REF' ) && class_exists( '\Presslabs\Cache\CacheHandler' ) ) {
+	if ( file_exists( WP_CONTENT_DIR . '/advanced-cache.php' ) ) {
```

Comment:
You can remove this starting from our 3.3 version

Reply:
Removed the advanced-cache loading as instructed.